dbt/models/pinval/docs.md

```diff
@@ -1,15 +1,14 @@
 # vw_assessment_card
 
-{% docs view_vw_assessment_card %}
-
+{% docs view_pinval_vw_assessment_card %}
```

Comment:
Even though this isn't exactly the same pattern that we usually use, since typically we leave the schema name out of the docs block variable name, I'm using it here since I'd like to start switching to this pattern eventually.

Reply:
Sounds good, so we want to move towards `tabletype_schemaname_tablename`?

Reply:
Yup, that's the goal!